precode.sh

```diff
@@ -6,38 +6,61 @@
 #   dir2
 #   dir3
 #       dir4
+mkdir /task
```

Comment:
1. Папки создаем относительно текущего скрипта, а не от корня файловой системы
2. Можно в одну строчку с флагом `-p`
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```